src/ui/components/Hero/index.js

```diff
@@ -11,15 +11,15 @@ import HeroSection from 'ui/components/HeroSection';
 import './styles.scss';
 
 
-type PropTypes = {|
+type Props = {|
   dispatch: Function,
   heroBanners: Object,
   name: string,
   random?: boolean,
-  sections: Array<React.Element<typeof HeroSection>>,
+  sections: Array<HeroSection>,
```

Comment:
why did you have to change it? Just curious. I thought they were interchangeable. I like how concise it is as `Array<HeroSection>`.

Reply:
I think it did not work well before.